### PR TITLE
[Woo POS][Cash Receipts] Reimplement the payment success screen as per changed design

### LINKED ISSUE #13129: [Woo POS][Cash & Receipts] Reimplement the payment success screen as per changed design
rNAclWFAdQRw7EXEcsNVai-fi-3248_6346

Image

### PR DESCRIPTION
**Pull request by kidinov**
Closes: #13129

### Description

The PR brings the changes to the UI of the payment success screen to match the changed design

### Testing information

* Pos -> complete the payment either with cash or card
* Notice the changed design
* Compare with Figma

rNAclWFAdQRw7EXEcsNVai-fi-3248_6346

### The tests that have been performed

Above

### Images/gif

https://github.com/user-attachments/assets/0b34f294-4e76-48c5-9856-4d65122b5b26

- [x] I have considered if this change warrants release notes and have added them to `RELEASE-NOTES.txt` if necessary. Use the "[Internal]" label for non-user-facing changes.

## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:

- [x] The PR is small and has a clear, single focus, or a valid explanation is provided in the description. If needed, please request to split it into smaller PRs.
- [x] Ensure Adequate Unit Test Coverage: The changes are reasonably covered by unit tests or an explanation is provided in the PR description.
- [x] Manual Testing: The author listed all the tests they ran, including smoke tests when needed (e.g., for refactorings). The reviewer confirmed that the PR works as expected on big (tablet) and small (phone) in case of UI changes, and no regressions are added.

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR is assigned to the milestone <a href="https://github.com/woocommerce/woocommerce-android/milestone/245">21.3</a>. This milestone is due in less than 2 days.<br />
Please make sure to get it merged by then or assign it to a milestone with a later deadline.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: WooCommerce/src/main/res/drawable/ic_woo_pos_check.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (samiuelson) [inline WooCommerce/src/main/res/drawable/ic_woo_pos_check.xml:1]: ❓ Can we delete the old icon `woo_pos_ic_payment_success.xml`?

💡 Also while we're here maybe let's unify drawable names – there are pos icons having names starting with `woo_pos_ic...` and `ic_woo_pos...`?

**Review by samiuelson — APPROVED** [review]
LGTM, @kidinov ! Please take a look at suggestions I added.

File: WooCommerce/src/main/res/drawable/ic_woo_pos_check.xml (hunk 1 above)
Comment (kidinov) [reply WooCommerce/src/main/res/drawable/ic_woo_pos_check.xml:1]: `woo_pos_ic_payment_success` - that was unused already before my change 🤔 . Removed that 

428949683243783198880fcad24287b974518003

> naming

Good call. Done here 5bef741d6741d8f40486b3081e6615be04610ec2

**Pull request merged**